src/content/reference/kernel-settings/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Tuned kernel settings
+description: Complete list of the tuned CoreOS kernel settings for Giant Swarm clusters.
+date: 2020-02-20
+layout: subsection
+weight: 100
+---
+
+# Kernel settings
+
+To properly run applications on a Giant Swarm cluster, we ensure that some kernel settings are tuned at the OS level. 
+
+## General Performance and Security settings
+
+| Setting                                | Value         | Description                                                         |
+| ---                                    | ---           | ---                                                                 |
+| kernel.kptr_restrict    	             | 2             | Hide kernel pointers to mitigate the kernels attack surface         |
+| kernel.sysrq   	                     | 0             | Reduce the kernel's attack surface                                  | 
+| net.ipv4.conf.all.log_martians  	     | 1             | Log all martians packets coming to existing network interfaces      |  
+| net.ipv4.conf.all.send_redirects  	 | 0             | Do not send redirects for IPv4                                      |
+| net.ipv4.conf.default.accept_redirects | 0             | Do not accept redirects for IPv4                                    |
+| net.ipv4.conf.default.log_martians     | 1             | Log all martians packets coming to freshly added network interfaces |
+| net.ipv4.tcp_timestamps                | 0             | Do not add timestamps to use less CPU cycles                        |
+| net.ipv6.conf.all.accept_redirects     | 0             | Do not send redirects for IPv6                                      |
+| net.ipv4.conf.default.accept_redirects | 0             | Do not accept redirects for IPv6                                    |       
```

Comment:
The name of the setting indicates that it is about IPv4. Is the description (IPv6) correct?

Reply:
My apologies, I made a mistake. I also reordered the fields if you want to take another look